### ISSUE TITLE
replays not visible

### ISSUE THREAD

**Issue by TuxNuX** [Lewdninja/liveme-pro-tools#55]
replays be on the site liveme.com but not be in liveme tools

same problem with other replays

![screenshot_20180505_130640](https://user-images.githubusercontent.com/17123357/39662661-a8bb6dd4-5065-11e8-9709-ba3b528f9534.png)
![screenshot_20180505_130545](https://user-images.githubusercontent.com/17123357/39662663-aed85b0a-5065-11e8-99bc-0a3a3831be33.png)

**Comment by kri2sa** [comment]
this happens with banned users. to watch the replay with the video player, if you have video id, you can use the old liveme-toolkit-

**Comment by rebuff10** [comment]
you are true i checked it too, but live me tools shows 0 replays on a banned user account and if you access that account on liveme.com it shows 1 replay only.For example i tried to access a banned account on liveme.com which has 80 replays on mobile app but it shows only 1 replay on liveme.com

**Comment by TuxNuX** [comment]
rebuff10 a, even on non-banned accounts on liveme.com it shows only one replay

but banned or not banned I do not understand why we see it on the official website and not in "liveme tools"

**Comment by ddpk** [comment]
@TuxNuX  It's pretty logical actually. It's just the liveme web developers being retarded. https://github.com/Lewdninja/liveme-pro-tools/issues/52#issuecomment-386905954 It's not just "one replay" its their latest replay, and not always meaning their latest broadcast. Since livemetools is accessing directly through API, it's not going to magically show the last one just because liveme.com is showing it. Livemetools is still trying to search all that user's replays through the API which access is blocked to when the user is banned, including through the mobile app. That's why nothing shows up. The reason it's shown on their web profile is a simple mistake. It probably has nothing to do with API. Consider that area on their profile simply automatically sharing the direct videoid link to every user's last replay. Not a feature that goes through their API I assume or i'm sure they would have blocked access to it by now for banned users.
